build.sbt

```diff
@@ -47,6 +47,8 @@ lazy val commonSettings = Seq(
     url("https://github.com/guardian/support-frontend"),
     "scm:git:git@github.com:guardian/support-frontend.git"
   )),
+  // https://www.scala-sbt.org/1.x/docs/Cached-Resolution.html
+  updateOptions := updateOptions.value.withCachedResolution(true)
```

Comment:
thanks for the link, this is described as experimental so how do you find the benefits? does it look like this will be made a full feature soon?

Reply:
I _think_ it's decreased the build times, but that could just be psychological 😅

Not sure when it will be made a full-feature; seems to be fairly prominent across guardian sbt projects. But also happy to remove.

Reply:
fair enough, not seen it before, keep it in if it's well used here! I just wanted to know more about the decision process.